include/apq/apq.h

```diff
@@ -1,3 +1,7 @@
 #pragma once
 
 #include <apq/connection_pool.h>
+
+namespace libapq {
```

Comment:
Why libapq?

Reply:
Name "apq" conflicts with the internal project name. Do you have any naming suggestions?

Reply:
`yapq`?

Reply:
is it really better? :)

Reply:
how about pqaxx for namespaces, libpqaxx for the project? Includes a not very subtle "axe"-pun in it... But the real reason I'm suggesting this is because I've googled around for other pq-libraries, and they all seem to be named libpq*xx: libpqxx, libpqmxx, libpq++...

Reply:
We do not provide adaptation of libpq. We provide an advanced convenient mechanism for asynchronous PostgreSQL queries. In this case, "apq" is the best name. But unfortunately, we have a project with this name. I don't like "lib" prefix, to be honest, and I thought about something like apq2, but such name will confuse users. I don't think what to name our project similar to the other projects is a good idea. In this case, a user can't easily distinguish this project from many other ones. It is better to have an original name.

How about "bpq - Better PostgreSQL Queries"? :) Or "acma_pq" with short namespace "acma" (sounds really good!)? Or "amq" - advanced mechanism [for] queries? Or even "pqnyan" - in this case, we can use pretty cat-mascot as a logo? =(^_^)=